src/client/Microsoft.Identity.Client/ApiConfig/Executors/ConfidentialClientExecutor.cs

```diff
@@ -57,6 +57,8 @@ public ConfidentialClientExecutor(IServiceBundle serviceBundle, ConfidentialClie
                 _confidentialClientApplication.AppTokenCacheInternal);
 
             requestParams.SendX5C = clientParameters.SendX5C;
+            requestParams.AutoDetectRegion = clientParameters.AutoDetectRegion;
```

Comment:
Do we need this flag in 2 places?

Reply:
Right we dont need it in request params. Removing it from there.